### PR TITLE
Feat/sprint9

### PR DESCRIPTION
**Pull request by lailabjil**

### CODE REVIEW

File: src/ui/components/ActivitySelecter/ActivitySelecter.tsx [hunk 1]
```diff
@@ -911,6 +914,17 @@ const nextStepMain = (
     } else nextClickCallback(false);
 };
 
+const navNextStep = (
+    value: string | undefined,
+    onChange: (isFullyCompleted: boolean, id?: string, suggesterId?: string, label?: string) => void,
+    nextClickCallback: (routeToGoal: boolean) => void,
+    routeToGoal: boolean,
+) => {
+    console.log(value);
```
Comment (MichaelC67) [inline src/ui/components/ActivitySelecter/ActivitySelecter.tsx:923]: Clean console log ?

File: src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx [hunk 2]
```diff
@@ -123,7 +123,7 @@ const CheckboxOneEdt = memo((props: CheckboxOneProps) => {
                     {label && (
                         <Box>
                             <Box className={classes.labelSpacer}>
-                                <label>{label}&nbsp;?</label>
+                                <h1 className={classes.h1}>{label}&nbsp;?</h1>
```
Comment (MichaelC67) [inline src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx:126]: Explain use of main title here (=>h1)

**Review by MichaelC67 — CHANGES_REQUESTED** [review]

File: src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx (hunk 2 above)
Comment (lailabjil) [reply src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx:126]: we have changed it due to two types of accessibility bugs.

We cannot have a label without being associated to a component by default + we must have a title per page.

As we currently have pages where we directly associate the page to these components, we have changed the labels (indirectly we had already conceived the question as a title) for titles.

Is there any problem for you that we have made this change?

Comment (MichaelC67) [reply src/ui/components/CheckboxOneEdt/CheckboxOneEdt.tsx:126]: No problem for me. We ll see with the expert in accessiblity. Thank you for the response.

**Pull request merged**